config/environments/development.rb

```diff
@@ -74,7 +74,16 @@
   config.amqp.requeue_key                 = "requeue.#{Rails.env}"
   config.amqp.main.queue                  = 'psd.mlwh'
   config.amqp.main.exchange               = 'psd.sequencescape'
-  config.amqp.main.routing_keys           = ['test.key']
+  config.amqp.main.routing_keys           = [
+    # Test binding for development
+    'test.key',
+    # Sequencescape bindings
+    'development.saved.sample.#',
+    'development.saved.study.#',
+    'development.message.#',
+    # Sample Extraction bindings
+    'development.activity.finished.#'
+  ]
   config.amqp.main.deadletter_exchange    = 'deadletters'
```

Comment:
Again, inspired by comments in the Readme. I've added the keys we use for Sequencescape and sample extraction. No real reason not to streamline this part of the setup. (Especially in development mode.)